<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>XScreenClass</class>
 <widget class="QWidget" name="XScreenClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>674</width>
    <height>350</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>XScreen</string>
  </property>
  <property name="windowIcon">
   <iconset resource="xscreen.qrc">
    <normaloff>:/XScreen/logo.ico</normaloff>:/XScreen/logo.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">#back
{
background-color:rgb(53,53,53);
border-radius:15px;
}
#all
{
background-color:rgb(35,35,35);
}
#title
{
color:rgb(0, 170, 255);
font:75 15pt &quot;黑体&quot;;
}
#timelabel
{
color:rgb(0, 170, 255);
font:75 20pt &quot;黑体&quot;;
}</string>
  </property>
  <widget class="QWidget" name="back" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>1000</height>
    </rect>
   </property>
   <widget class="QPushButton" name="minButton">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>0</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/XScreen/mini.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="closeButton">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>0</y>
      <width>31</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/XScreen/close.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QWidget" name="all" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>40</y>
      <width>671</width>
      <height>311</height>
     </rect>
    </property>
    <widget class="QPushButton" name="recordButton">
     <property name="geometry">
      <rect>
       <x>550</x>
       <y>30</y>
       <width>101</width>
       <height>91</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton:!hover
{
	background-image: url(:/XScreen/record_normal.png);
}

QPushButton:hover
{
	background-image: url(:/XScreen/record_hot.png);
}

QPushButton:pressed
{
	background-image: url(:/XScreen/record_pressed.png);
	background-color: rgba(255, 255, 255, 0);
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="urlEdit">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>50</y>
       <width>481</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>D:\video</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="widthEdit">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>210</y>
       <width>71</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>1280</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="heightEdit">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>210</y>
       <width>71</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>720</string>
     </property>
    </widget>
    <widget class="QLabel" name="timelabel">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>140</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>000:00</string>
     </property>
    </widget>
    <widget class="QLabel" name="infolabel">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>249</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>维护：夏佳权</string>
     </property>
    </widget>
    <widget class="QLabel" name="infolabel_2">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>210</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>帧率</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="fpsEdit">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>210</y>
       <width>71</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>10</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="title">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>10</y>
      <width>261</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>XScreen 屏幕录像机 v1.0</string>
    </property>
   </widget>
   <widget class="QLabel" name="logo">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/XScreen/logo.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="xscreen.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>closeButton</sender>
   <signal>clicked()</signal>
   <receiver>XScreenClass</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>657</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>660</x>
     <y>363</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>minButton</sender>
   <signal>clicked()</signal>
   <receiver>XScreenClass</receiver>
   <slot>showMinimized()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>627</x>
     <y>23</y>
    </hint>
    <hint type="destinationlabel">
     <x>618</x>
     <y>377</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>recordButton</sender>
   <signal>clicked()</signal>
   <receiver>XScreenClass</receiver>
   <slot>Record()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>599</x>
     <y>132</y>
    </hint>
    <hint type="destinationlabel">
     <x>594</x>
     <y>436</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>Record()</slot>
 </slots>
</ui>
